sklearn/utils/estimator_checks.py

```diff
@@ -238,17 +247,30 @@ def check_estimator(Estimator):
     will be run if the Estimator class inherits from the corresponding mixin
     from sklearn.base.
 
+    This test can be applied to classes or instances.
+    Classes currently have some additional tests that related to construction,
+    while passing instances allows the testing of multiple options.
+
     Parameters
     ----------
     Estimator : class
```

Comment:
I also think you should rename to `estimator` to emphasise that instance is now the norm. I think it's very unlikely that that will break code.